core/src/main/java/org/elasticsearch/index/mapper/MapperService.java

```diff
@@ -129,9 +134,9 @@ public MapperService(IndexSettings indexSettings, AnalysisService analysisServic
             "\"_default_\":{\n" +
                 "\"properties\" : {\n" +
                     "\"query\" : {\n" +
-                        "\"type\" : \"object\",\n" +
-                        "\"enabled\" : false\n" +
-                    "}\n" +
+                        "\"type\" : \"percolator\"\n" +
+                    "},\n" +
+                    "\"" + QueryMetadataService.QUERY_METADATA_FIELD + "\" : { \"type\" : \"string\", \"index\" : \"not_analyzed\" }" +
```

Comment:
now that there is a percolator type maybe we don't need it at all? Ie the percolator field mapper would manage its own sub fields like the geo point field mapper does for its lat/lon/geohash fields